src/pkgcheck/bash/__init__.py

```diff
@@ -92,8 +94,10 @@ def build_library(output_path, repo_paths):  # pragma: no cover
         build_library(lib, [bash_src])
 
 
-if os.path.exists(lib):
-    lang = Language(lib, 'bash')
+syslib = find_library('tree-sitter-bash')
+
+if syslib or os.path.exists(lib):
+    lang = Language(syslib if syslib else lib, 'bash')
```

Comment:
I think

```suggestion
    lang = Language(syslib or lib, 'bash')
```

will work too.

Reply:
Neat! Thanks.